src/helpers/cmdliner_ext.ml

```diff
@@ -0,0 +1,48 @@
+include Cmdliner
+
+(* TODO: use log label for something *)
+let setup_log _log_label json_logs style_renderer level =
+  (match style_renderer with
+  | Some style_renderer -> Fmt_tty.setup_std_outputs ~style_renderer ()
+  | None -> Fmt_tty.setup_std_outputs ());
+  Logs.set_level level;
+
+  (match json_logs with
+  | true -> Logs.set_reporter (Json_logs_reporter.reporter Fmt.stdout)
+  | false -> Logs.set_reporter (Logs_fmt.reporter ()));
+
+  (* disable all non-deku logs *)
+  (* TODO: this should only be disabled in the printf output, not in the output where we write to a file, so this is just a temporary measure *)
+  List.iter
+    (fun src ->
+      let src_name = Logs.Src.name src in
+      if
+        (not (String.starts_with ~prefix:"deku" src_name))
+        && not (String.equal src_name "application")
+      then
+        Logs.Src.set_level src (Some Logs.Error))
+    (Logs.Src.list ())
+
+let setup_log =
+  let open Term in
+  let log_label =
+    let docv = "Log label" in
+    let doc = "This will be prepended to all logs." in
+    Arg.(value & opt (some string) None & info ["log-label"] ~docv ~doc) in
+  let json_logs =
+    let docv = "Json logs" in
+    let doc = "This determines whether logs will be printed in json format." in
+    Arg.(value & flag & info ~doc ~docv ["json-logs"]) in
```

Comment:
@ulrikstrid is this enough for JSON logging, or do you need an environment variable too?

Reply:
We can pass flags when running in kubernetes so this is perfect :ok_hand: